geolocation.py

```diff
@@ -6,8 +6,60 @@
     CODE_COUNTRY_DICT,
     STATE_ABBREV,
     STATE_NAMES,
+    CITY_COUNTRY_STRINGS,
 )
 
+def levenshteinDistance(s1, s2):
```

Comment:
@kdobolyi, I love your custom writing of a Levenshtein distance. That's how everyone knows you're legit. There is a python package (python-levenshtein) for this, but writing it from scratch is certainly in line with the zeitgeist of our project, so as long as you want the from-scratch version, yes, let's go for it!

Reply:
I don't have a strong opinion either way, but to be fair I copied the code from stackoverflow, so let me at least cite it, thanks!